pkg/tfgen/main.go

```diff
@@ -137,6 +141,12 @@ func newTFGenCmd(pkg string, version string, prov tfbridge.ProviderInfo) *cobra.
 				return err
 			}
 
+			// Exporting collected coverage data to a given directory if the COVERAGE_OUTPUT_DIR env is set
+			COD := os.Getenv("COVERAGE_OUTPUT_DIR")
```

Comment:
we can actually do something like

```
cod, exists := os.LookupEnv("COVERAGE_OUTPUT_DIR")
if exists {
   coverageTracker.exportResults(cod)
}
```

then you don't need to check for an empty string

Reply:
Fixed